crates/wassette/src/lib.rs

```diff
@@ -100,56 +103,143 @@ pub struct ValidationStamp {
     pub content_hash: Option<String>,
 }
 
-#[derive(Debug, Default)]
```

Comment:
Nit: This is _technically_ a breaking change if you remove this, but not one I really care about. I do think for debug purposes, it would be nice to still print the `tool_map` and `component_map` in a manually implemented debug